### DISCUSSION TITLE
。。。

### DISCUSSION THREAD

**Discussion by cyq20091029** [share121/inter-knot#2778] [其他]

**Comment by Nie7bai** [comment]
节奏之类的话感觉还是少发在绳网里了，现在这里还比较和谐，不希望变得和B站一样全是讨论节奏的<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >

**Reply by cyq20091029** [reply]
好的，删除了

**Comment by cx525** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/07/11/38d924d952f0e674f5ce7580890cbc65_3662132904829338895.png" alt="刃 锁定" width="75" >
谁把史搬我脸上，杀了
<img src="https://bbs-static.miyoushe.com/static/2023/07/11/38d924d952f0e674f5ce7580890cbc65_3662132904829338895.png" alt="刃 锁定" width="75" >

**Reply by cyq20091029** [reply]
好的，删除了

**Comment by Nie7bai** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/07/31/de6cf640b821a169d3cfb52621cec21b_242979188329737974.png" alt="椒丘 辣" width="75" >.
